### PR TITLE
Resolve local notification problem(Android).

### PR DESCRIPTION
**Pull request by MechanicKim**
- Updated react-native-fcm
- Referred react-native-fcm git page and modified manifest file.

I tested app and confirmed that local notification worked correctly.

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -75,16 +80,6 @@
...
-        <receiver android:name="com.evollu.react.fcm.FIRLocalMessagingPublisher"/>
-        <receiver android:enabled="true" android:exported="true"  android:name="com.evollu.react.fcm.FIRSystemBootEventReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED" />
-                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
-                <action android:name="com.htc.intent.action.QUICKBOOT_POWERON" />
-                <category android:name="android.intent.category.DEFAULT" />
-            </intent-filter>
-        </receiver>
     </application>
```
Comment (ifndefdeadmau5) [inline android/app/src/main/AndroidManifest.xml:83]: Could you explain for me what's going on here?? because I have no knowldege about this

Comment (MechanicKim) [reply android/app/src/main/AndroidManifest.xml:83]: You can refer this page
[react-native-fcm](https://github.com/evollu/react-native-fcm)
The way of setting is changed.

Comment (MechanicKim) [reply android/app/src/main/AndroidManifest.xml:83]: To use notification, setting in manifest file is necessary(To register FCM service).

**Comment by ifndefdeadmau5** [comment]
Thanks LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/150/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**